test/ServerSpec.js

```diff
@@ -96,50 +109,76 @@ describe('Unprotected routes: ', function() {
       });
     });
 
-    it('should return empty response for nonexistent users', function(done) {
-      request(loginOptions, function(err, res, body) {
+  });
+
+  describe('Memory creation', function() {
+   
+    // note: need aws credentials set up (in the current tab) for this to work
+    it ('should create a local file and push it to s3', function(done) {
```

Comment:
If there is no error, it means that s3 uploaded successfully, but in the future we could definitely add a test to make sure that s3 actually has the photo. The response object is not the id of the memory, so it may be difficult to test that though (since we don't know which id to check for).